packages/astro/src/assets/runtime.ts

```diff
@@ -0,0 +1,54 @@
+import {
+	createComponent,
+	render,
+	spreadAttributes,
+	unescapeHTML,
+} from '../runtime/server/index.js';
+import type { ImageMetadata } from './types.js';
+import { makeNonEnumerable, normalizeProps } from './utils/svg.js';
```

Comment:
Can we inline both of these function into this file? It looks to only be used here. That way we don't import `svg.js` which may bundle `ultrahtml` into the SSR runtime, or in the future may accidentally bundle other things in that file.